packages/react-dom/src/events/ReactDOMUpdateBatching.js

```diff
@@ -102,27 +87,8 @@ export function discreteUpdates(fn, a, b, c, d) {
   }
 }
 
-let lastFlushedEventTimeStamp = 0;
-export function flushDiscreteUpdatesIfNeeded(timeStamp: number) {
-  // event.timeStamp isn't overly reliable due to inconsistencies in
-  // how different browsers have historically provided the time stamp.
-  // Some browsers provide high-resolution time stamps for all events,
-  // some provide low-resolution time stamps for all events. FF < 52
-  // even mixes both time stamps together. Some browsers even report
-  // negative time stamps or time stamps that are 0 (iOS9) in some cases.
-  // Given we are only comparing two time stamps with equality (!==),
-  // we are safe from the resolution differences. If the time stamp is 0
-  // we bail-out of preventing the flush, which can affect semantics,
-  // such as if an earlier flush removes or adds event listeners that
-  // are fired in the subsequent flush. However, this is the same
-  // behaviour as we had before this change, so the risks are low.
-  if (
-    !isInsideEventHandler &&
-    (!enableDeprecatedFlareAPI ||
-      timeStamp === 0 ||
-      lastFlushedEventTimeStamp !== timeStamp)
```

Comment:
Can you remind me why we no longer care about time stamps? Why was this Flare-specific?

Reply:
It was Flare specific, but we could re-include it. Some of the SimpleEventPlugin tests might fail again though.

Reply:
I'm not arguing to change this, but more just curious why we only enabled it for Flare listeners when batching itself is generic.

Reply:
With Flare, we dispatched events again if they were discrete, meaning in some cases we'd over-flush and it causes UI issues internally. Given we've moved back to only handling the flushing on incoming discrete events, then we don't necessarily need this behavior anymore. I can see how we might want to re-introduce it in the future though, maybe one for @sebmarkbage to think about when he revists some of the flushing related work.